docs/user-docs/latest/settings/audit-logs.md

```diff
@@ -8,10 +8,24 @@ canonicalUrl: https://docs.strapi.io/user-docs/latest/settings/audit-logs.html
 
 The **Audit Logs** section provides a <!--searchable and filterable--> display of all activities performed by users of the Strapi application.
 
-They are accessible from the **Administrator Panel** section of the [**Settings**](./managing-global-settings.md) panel.
+Audit logs are only accessible to users with the **Super Admin** role by default. They are available in the **Administrator Panel** section of the [**Settings**](./managing-global-settings.md) panel.
 
 ![Audit Logs panel](../assets/settings/settings_audit-logs.png)
 
+## Events logged
+
+The following events are logged:
+
+| Event | Actions |
+| --- | --- |
+| Content Type | Create, update, delete |
```

Comment:
To me, it looks a little off to start the lists with capital letters, but I am unsure of the "right" way to do this.